Description: Build Screen without NVidia 3D Vision (libnvstusb) support
 Screen otherwise dlopen()s a prebuilt PsychPlugins/libnvstusb.so, which is
 a binary blob that is deleted in postPatch.
Origin: Ported from Debian deb_no_PTB_USE_NVSTUSB (3.0.22.3+dfsg-4).
---
--- a/PsychSourceGL/Source/linuxmakeitoctave3.m
+++ b/PsychSourceGL/Source/linuxmakeitoctave3.m
@@ -89,7 +89,7 @@
 
     % Build against system installed GStreamer-1.8+, ideally 1.18+.
     iflags = pkgconfig_iflags('gstreamer-1.0 gstreamer-app-1.0 gstreamer-video-1.0 gstreamer-pbutils-1.0 glib-2.0 libxml-2.0');
-    mex("-W -std=gnu99 -Wno-stringop-truncation", "--output", "../Projects/Linux/build/Screen.mex", "-Wno-date-time", "-DPTBMODULE_Screen", "-DPTB_USE_GSTREAMER", "-DPTBVIDEOCAPTURE_LIBDC", "-DPTB_USE_NVSTUSB", "-DPTBOCTAVE3MEX", "-D_GNU_SOURCE", "-I../Cohorts/libnvstusb-code-32/include", "-ICommon/Base", "-ICommon/Screen", "-ILinux/Base", "-ILinux/Screen", iflags{:}, "Linux/Base/*.c", "Linux/Screen/*.c", "Common/Screen/*.c", "Common/Base/*.c", "Common/Screen/tinyexr.cc", "-lc", "-ldl", "-lrt", "-lGL", "-lGLU", "-lGLEW", "-lX11", "-lXext", "-lX11-xcb", "-lxcb", "-lxcb-dri3", "-lxcb-present", "-lgstreamer-1.0", "-lgstbase-1.0", "-lgstapp-1.0", "-lgstvideo-1.0", "-lgstpbutils-1.0", "-lgobject-2.0", "-lgmodule-2.0", "-lxml2", "-lgthread-2.0", "-lglib-2.0", "-lXxf86vm", "-ldc1394", "-lusb-1.0", "-lpciaccess", "-lXi", "-lXrandr", "-lXfixes", "-lXcomposite");
+    mex("-W -std=gnu99 -Wno-stringop-truncation", "--output", "../Projects/Linux/build/Screen.mex", "-Wno-date-time", "-DPTBMODULE_Screen", "-DPTB_USE_GSTREAMER", "-DPTBVIDEOCAPTURE_LIBDC", "-DPTBOCTAVE3MEX", "-D_GNU_SOURCE", "-ICommon/Base", "-ICommon/Screen", "-ILinux/Base", "-ILinux/Screen", iflags{:}, "Linux/Base/*.c", "Linux/Screen/*.c", "Common/Screen/*.c", "Common/Base/*.c", "Common/Screen/tinyexr.cc", "-lc", "-ldl", "-lrt", "-lGL", "-lGLU", "-lGLEW", "-lX11", "-lXext", "-lX11-xcb", "-lxcb", "-lxcb-dri3", "-lxcb-present", "-lgstreamer-1.0", "-lgstbase-1.0", "-lgstapp-1.0", "-lgstvideo-1.0", "-lgstpbutils-1.0", "-lgobject-2.0", "-lgmodule-2.0", "-lxml2", "-lgthread-2.0", "-lglib-2.0", "-lXxf86vm", "-ldc1394", "-lusb-1.0", "-lpciaccess", "-lXi", "-lXrandr", "-lXfixes", "-lXcomposite");
 
     unix(['cp ../Projects/Linux/build/Screen.mex ' PsychtoolboxRoot target]);
 end
